pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -264,8 +260,9 @@ var (
 			KubeletConfigFileContent: "",                   // s
 			KubeletFlags:             map[string]string{},  // psX
 		},
-		MessageOfTheDay: "",    // td
-		IsKata:          false, // n
+		TlsBootstrappingConfig: &nbcontractv1.TLSBootstrappingConfig{},
```

Comment:
why not EnableSecureTlsBootstrapping = false, TlsBootstrapToken = "" and CustomSecureTlsBootstrapAppserverAppid = ""?

Reply:
Didn't think necessary since the default values will be empty string anyway, but I can add them back

Reply:
ya I think adding them explicitly could save us some time for debugging in the future, as we may have `no presence` and `explicit presence` cases from the contract. For example, in this case, the initial value for EnableSecureTlsBootstrapping is nil (because with optional label), but initial values for TlsBootstrapToken and CustomSecureTlsBootstrapAppserverAppid are "". Thanks.